.travis.yml

```diff
@@ -54,6 +54,28 @@ matrix:
       env:
         - MATRIX_EVAL="CXX=g++-7"
 
+    # Lint Python 2 with flake8
+    - os: linux
```

Comment:
I don't think we need to keep Python 2 support. Yes, some of the algorithms contain Python 2 code, but they _should_ have been Python 3 in the first place. I would remove this matrix entry.